python/looker/rtl/user_session.py

```diff
@@ -61,48 +52,58 @@ def get_token(self):
             self.login()
         return self._token
 
-    def authenticate(self):
-         # TODO: still to be built based on request
```

Comment:
this will be one of many questions to bring up (or maybe we'll resolve it ourselves as we flesh out the implementation more)